src/main/java/seedu/contax/logic/commands/BatchCommand.java

```diff
@@ -84,8 +110,17 @@ public CommandResult execute(Model model) throws CommandException {
         return new CommandResult(resultOutput.toString());
     }
 
-    private List<Index> matchInputStringToIndex(Model model, SearchType searchType, String userInput)
-            throws CommandException {
+    /**
+     * Matches base on user input value and return the index list that matched the condition given.
+     *
+     * @param model         model passed from execution
+     * @param searchType    the field that user search to compare for
+     * @param userValue     user input value
+     * @param matchType     equals, start or end with
+     * @return              list of index that match given condition
+     */
+    private List<Index> matchInputStringToIndex(Model model,
+                                                SearchType searchType, String userValue, String matchType) {
```

Comment:
Likewise, matchType should be an Enum rather than a String.

Reply:
thx will change